content/ react-security-vulnerabilities/index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "React Security Vulnerabilities and How to Fix/Prevent them"
+date: "2021-12-21"
+coverImage: "react-security.png"
+author: "Gaurav Kr. Roy"
+tags: ["React", "vulnerability", "prevent", "security"]
+description: "React security vulnerabilities are difficult to detect with naked eyes. However, it can be disastrous for progressive web apps if developers don't know how to fix them."
+---
+
+All React developers love to leverage the benefits React caters to in developing web applications. But developers need to keep in mind the security postures while creating React web apps. React applications face a vast attack surface and are prone to different vulnerabilities. This article is a checklist of React security best practices that every developer should know before diving into PWA (progressive web application) development. 
+If you are new to progressive web applications and React, let's get familiar with these terminologies first.
+
+## What are Progressive Web Applications (PWA)?
+Progressive Web Apps (PWA) are apps built using web technologies like HTML, CSS, and JavaScript (JS). But these apps deliver the experience, feel, and functionality of a native app. PWA combines new technologies and integrations to build a reliable, engaging, accessible, and secure application. Developers mostly use React on top of HTML and JavaScript to build a progressive web app.
+
+## What is React?
+React is popular among progressive web app developers. This open-source, robust JavaScript library helps in building user interfaces based on UI components. React gains popularity in the software development industry because it allows developers to create lightweight apps with additional facilities: security, push notification, app-like look and feel, etc. Some popular companies that have become the early adopters of React are Instagram, Netflix, Airbnb, Uber Eats, Discord, the New York Times, etc. 
+
+## Security Vulnerabilities in React and how to prevent them –

```

Comment:
Remove `–` at the end of all the titles. And use consistent title case for sub-heading throughout the article